source/includes/_08-additional-features.md.erb

```diff
@@ -309,10 +309,10 @@ app.post("/search", (req, res) => {
         </ul>
         
         For demo searches, charges may be returned, but they will not be deducted
```

Comment:
Remove comma: "...charges may be returned but they will not..."

Reply:
👍